beta/src/pages/learn/start-a-new-react-project.md

```diff
@@ -1,41 +1,41 @@
 ---
-title: Start a New React Project
+शीर्षक: एक नया React शुरू करें
 ---
 
 <Intro>
 
-If you're learning React or considering adding it to an existing project, you can get started quickly by [adding React to any HTML page with script tags](/learn/add-react-to-a-website). If your project will need many components and many files, it might be time to consider the options below!
+यदि आप React सीख रहे हैं या इसे किसी मौजूदा प्रोजेक्ट में जोड़ने पर विचार कर रहे हैं, तो आप [स्क्रिप्ट टैग के साथ किसी भी एचटीएमएल पेज पर रिएक्ट जोड़कर](/learn/add-react-to-a-website) शुरू कर सकते हैं। यदि आपकी परियोजना को कई घटकों और कई फाइलों की आवश्यकता होगी, तो यह नीचे दिए गए विकल्पों पर विचार करने का समय हो सकता है!
```

Comment:
`प्रोजेक्ट में जोड़ने पर विचार कर रहे हैं` => `प्रोजेक्ट में इस्तेमाल करने पर विचार कर रहे हैं`

Reply:
`एचटीएमएल` => `HTML` we are not translating this word

Reply:
`रिएक्ट` => `React`

we are not translating React word

Reply:
`स्क्रिप्ट टैग के साथ किसी भी एचटीएमएल पेज पर रिएक्ट जोड़कर` => `स्क्रिप्ट टैग को किसी भी HTML पेज पर React इस्तेमाल करके`

Reply:
`आपकी परियोजना` => `आपके प्रोजेक्ट`

Reply:
`घटकों` => `कौम्पोनॅन्ट`

Reply:
`विकल्पों` => `औपशंस`

vikalp sounds a bit complex. I can't think of any other word. let me you if you have any suggestions.